Automattic/jetpack
# Social: Render meta tags on singular posts/pages

This PR adds Open Graph and Twitter Cards meta tags to singular posts and pages. It's a basic but well-tested implementation—in the near future we should aim to consolidate this with the extra functionality that the Jetpack plugin provides with regards to meta tags.

#### Changes proposed in this Pull Request:

* Add Open Graph and Twitter Cards meta tags to singular posts and pages.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make sure all the tests pass.
* Create a new post or page, and make sure there's no `og:image` and `twitter:image` tags present in the source.
* Add a featured image (more than 200 pixels in width and height) and check that the tags are there.
* Replace this featured image with a smaller one (<200 pixels width or height) and confirm the tags are gone.
* Switch back to the original featured image, but add a password to the page. Check that the tags are gone.
